src/components/UserInfo.js

```diff
@@ -1,16 +1,30 @@
 export default class UserInfo {
-  constructor(title, subtitle) {
+  constructor(title, subtitle, avatarSelector) {
     this.title = document.querySelector(title);
     this.subtitle = document.querySelector(subtitle);
+    this._profileAvatarElement = document.querySelector(avatarSelector)
   }
   getUserInfo() {
     return {
       title: this.title.textContent,
       subtitle: this.subtitle.textContent,
     };
   }
-  setUserInfo({ title, subtitle }) {
+  setUserInfo({ title, subtitle, avatar }) {
     this.title.textContent = title;
     this.subtitle.textContent = subtitle;
+    this._profileAvatarElement.src = avatar;
```

Comment:
There is no need to change the avatar here, since it's controlled by a separate form. You can call two methods when initializing the user info

Reply:
Done

Reply:
Please remove line 16 and use `setAvatarInfo` method in index.js instead